.github/workflows/pr-check.yml

```
@@ -96,12 +96,10 @@ jobs:
         \( -name '*.sh' -o -name "*.go" -o -name "*.yaml" -o -name "*.yml" \))
         LICENSE_TEMPLATE="${GITHUB_WORKSPACE}/hack/license-header.txt"
         check-license-header -f "${LICENSE_TEMPLATE}" ${FILES_TO_CHECK_LICENSE}
-  dependencies-md-validation:
+  dependencies-validation:
     runs-on: ubuntu-22.04
```

nit, use latest version of ubuntu runner, either use `ubuntu-latest` or `24.04`
```suggestion
    runs-on: ubuntu-24.04
```